src/features/system/command.js

```diff
@@ -77,17 +78,19 @@ async function parseFeature(message, request) {
 
 async function parseArguments(message, request) {
     let rawargs = message.content.slice(`${request.prefix}${request.feature.name} `.length)
+    const separator = rawargs.indexOf(' ') < rawargs.indexOf('\n') ? ' ' : '\n'
```

Comment:
In case when admin uses command without space in the end of lane, command will not be recognized (parsed like `$likeroles\n`